debug/org.eclipse.cdt.debug.ui/plugin.properties

```diff
@@ -282,3 +282,10 @@ popup.stepIntoSelection.name=Step Into Selection
 DebuggerConsoleView.name=Debugger Console
 
 GenericTarget.name=Generic Target
+
+# Toggle Instruction Step Mode Command Activity support
+org.eclipse.cdt.debug.ui.category.name=C/C++ Debug UI
+org.eclipse.cdt.debug.ui.category.description=Controls visibility of C/C++ Debug UI contributions
+
+org.eclipse.cdt.debug.ui.activity.toggleInstructionStepMode.name=Toggle Instruction Step Mode Command
+org.eclipse.cdt.debug.ui.activity.toggleInstructionStepMode.description=Activity to control visibility of Toggle Instruction Step Mode command
```

Comment:
Probably better: `Control visibility of 'Toggle Instruction Step Mode'  button on the main toolbar`